src/app/pages/plans/plan/plan.page.html

```diff
@@ -0,0 +1,48 @@
+@if (itemMetadata(); as info) {
+  <!--<app-static-map-image [location]="location()"></app-static-map-image>-->
+  <ion-header>
+    <ion-toolbar appHeaderColor>
+      <ion-buttons slot="start">
+        <ion-back-button text="" defaultHref="/plans"></ion-back-button>
+      </ion-buttons>
+      <ion-title> {{ info.name }} </ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content>
+    <div class="content">
+      <div class="tag-row">
+        <nve-tag saturation="default" variant="neutral">
+          <nve-icon slot="prefix" src="assets\icon\calendar_today.svg" />
+          <span>{{ "PLANS.DATE_EDITED" | translate }}</span> <span slot="extra">{{ info.date | date: "short" }}</span>
+        </nve-tag>
+        <nve-tag saturation="default" variant="neutral">
+          <nve-icon slot="prefix" src="assets\icon\altitude.svg" />
+          <span slot="extra">-- {{ "MAP_CENTER_INFO.ABOVE_SEA_LEVEL" | translate }}</span>
+        </nve-tag>
+        <nve-tag saturation="default" variant="neutral">
+          <nve-icon slot="prefix" src="assets\icon\arrow_range.svg" />
+          <span slot="extra">{{ info.lengthKm ? `${info.lengthKm.toFixed(2)} km` : '-- km' }}</span>
+        </nve-tag>
+      </div>
+      <form>
+        <nve-checkbox [checked]="visibleOnMap()" (sl-change)="onVisibleOnMapChange($event)">
+          {{ "PLANS.VISIBLE_ON_MAP" | translate }}
+        </nve-checkbox>
+        <nve-input [value]="name()" [label]="'PLANS.NAME' | translate" (sl-change)="onNameChange($event)"></nve-input>
+        <nve-textarea
+          [value]="comment()"
+          (sl-change)="onCommentChange($event)"
+          [label]="'PLANS.COMMENT' | translate"
+        ></nve-textarea>
+        <div class="button-group">
+          <nve-button variant="primary" type="submit" (click)="onSave()">Lagre</nve-button>
```

Comment:
Dette med egen lagre-knapp per skjema er noe nytt i appen. Selv syns jeg ikke brukeropplevelsen er så god nå, med lagreknappen. Jeg prøvde feks å toggle en tur på bare med togglen, men glemte å trykke lagre fordi det er uvant og litt uvanlig sammenligna med appen ellers.

Tenker vi bør skrive om skjemaet til å lagre oppdateringene fortløpende så vi kan fjerne lagre-knappen. Eventuelt burde vi kanskje hatt to visninger av skjemaet, for å gjøre det tydeligere at "nå redigerer du". Feks at det var en helt statisk visning uten inputs / toggle, men med en rediger-knapp som åpner redigeringsvisning.